### PR TITLE
feat:Add XUI support (FPL-994)

### PR DESCRIPTION
**Pull request by tomaszpowroznik**
https://tools.hmcts.net/jira/browse/FPLA-994

### CODE REVIEW

File: compose/defaults.conf [hunk 1]
```diff
@@ -3,4 +3,5 @@ frontend
 sidam
 sidam-local
 sidam-local-ccd
-elasticsearch
+xui
+fpl
```
[the remark this answers is not in the document]
Comment (tomaszpowroznik) [reply compose/defaults.conf:7]: of course, i had this overrode in .tag.env with dm-store and had false confirmation by e2e that everything works

File: compose/xui.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+version: '2.4'
+
+services:
+  xui:
+    image: "hmctspublic.azurecr.io/xui/webapp:latest"
+    environment:
+      MICROSERVICE: xui_webapp
+      PROTOCOL: http
+      XUI_ENV: local
```
[the remark this answers is not in the document]
Comment (tomaszpowroznik) [reply compose/xui.yml:10]: see https://devops.stackexchange.com/questions/537/why-is-one-not-allowed-to-use-a-boolean-in-a-docker-compose-yml

File: mocks/wiremock/__files/organisationMockResponse.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "organisationIdentifier": "62LYJRF",
+  "name": "Local authority name",
+  "status": "ACTIVE",
+  "sraRegulated": false,
+  "superUser": {
+    "firstName": "Tomasz",
+    "lastName": "Powroznik",
```
[the remark this answers is not in the document]
Comment (tomaszpowroznik) [reply mocks/wiremock/__files/organisationMockResponse.json:8]: copied file from fpl-ccd-configuration, did not even know that my name is there :)

File: compose/xui.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (akomisarek) [reply compose/xui.yml:10]: Thanks.

**Pull request merged**